configure.js

```diff
@@ -13,17 +13,20 @@ const debug = require('debug')('action-dashboard:configure');
 
 const bodyParser = require('body-parser')
 const routes = require('./routes')
+const middleware = require('./webhooks')
 
 module.exports = {
     before: (app) => {
         app.use(bodyParser.json());
         app.use('/api', routes);
+
```

Comment:
What does moving this here enable?

Reply:
It is not possible to open multiple ports on Heroku. So instead of sending the hooks to a specific port, i added a route where Guthub can send the hooks to.